libs/common/grpc.go

```diff
@@ -136,7 +136,7 @@ func loggingUnaryInterceptor(ctx context.Context, req interface{}, info *grpc.Un
 		message := statusError.Message()
 		details := statusError.Details()
 		log.
-			Warn().
+			Error().
```

Comment:
this is not the expected behavior
> *Some* errors, notably those that are Internal, should be logged on the Error level.

Users may enter invalid input, or all kind of other issues may arise, which are not worthy being an Error.
But if the route handler deems the issue to be caused by an issue within the service (i.e. returning an InternalServerError), it should be an Error, not just a warning.

Reply:
Okay, get it. I've updated the code to resolve the error level by specific errors.